build.py

```diff
@@ -146,6 +139,7 @@ def run_stage(stage: Path, environment, build_dir):
         "IMAGE_OUTPUT_PATH": str(args.output_file),
         "BUILD_DIR": str(args.build_dir),
         "CACHE_DIR": str(args.cache_dir),
+        "PLATFORM": str(args.platform),
```

Comment:
```suggestion
        "PLATFORM": args.platform,
```

`args.platform` should already be a string. We have to run `str` on `args.build_dir` and `args.cache_dir` because they are parsed to `Path` objects by `argparse`.